BedrockServer.cpp

```diff
@@ -40,9 +40,20 @@ void BedrockServer::acceptCommand(SQLiteCommand&& command, bool isNew) {
             command.writeConsistency = SQLiteNode::QUORUM;
             SINFO("Forcing QUORUM consistency for command " << command.request.methodLine);
         }
-        SINFO("Queued new '" << command.request.methodLine << "' command from bedrock node, with " << _commandQueue.size()
-              << " commands already queued.");
-        _commandQueue.push(BedrockCommand(move(command)));
+
+        // If this is a status or a contorl command we need to handle it here, or else
+        // it will get stuck in the _commandQueue if we are in `Detached` mode and the
+        // command comes from a bedrock node.
+        SData request;
+        request.deserialize(command.request.content);
+        request.methodLine = command.request.methodLine;
+        BedrockCommand newCommand(request);
+        if (!_handleIfStatusOrControlCommand(newCommand)) {
+            SINFO("Queued new '" << command.request.methodLine << "' command from bedrock node, with " << _commandQueue.size()
+                    << " commands already queued.");
+            _commandQueue.push(BedrockCommand(move(command)));
```

Comment:
We've already constructed a new command. How about:
```
_commandQueue.push(move(newCommand));
```

Reply:
As discussed offline, i'll just expose attach/detach to plugins set _detach() and back out this change so we stop double deserializing commands that come through here.